templates/jobs/jobs.html

```diff
@@ -50,6 +52,7 @@
                                             {{ job.title }}
                                         </a>
                                     </td>
+                                    <td>{{ job.cities }}, {{ job.country }}</td>
```

Comment:
This could end up being a comma separated list of lots of cities followed by country.  If we're going with restricting submitters to one per country, this might be better formated as
```
{{ job.country }}: {{ job.cities }}
```
or 
```
{{ job.cities }} ({{ job.country }})
```